integrations/azure/README.md

```diff
@@ -0,0 +1,44 @@
+# Azure ORT starter pipeline
+
+The starter pipeline allows you to easily run an ORT scan on Azure DevOps.
+
+To run the pipeline, simply:
+
+- create a new YAML-based pipeline in your Azure DevOps project based on the pipeline file in this repository
+- run the pipeline with the preferred set of parameters
+
```

Comment:
Can you add a section with a list of the various files included in /azure and explain in one sentence what each file does?